data/core.yaml

```
@@ -786,6 +786,8 @@ en:
       inch: in
     max_length_reached: "This string is longer than the maximum length of {maxChars} characters. Anything exceeding that length will be truncated."
     set_today: "Sets the value to today."
+    set_mapillary: "Sets the current ID of mapillary"
+    show_mapillary_from_field: "Show image on viewer"
```

```suggestion
    set_photo_from_viewer: "Store this photo on the currently selected map object"
    set_photo_from_field: "Use the currently displayed photo"
    show_photo_from_field: "Open image in viewer"
```

(the string references would of course need to be adjusted accordingly as well)